code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -311,3 +311,23 @@
 	new /obj/item/weapon/spacecash/bundle/c1000(src)
 	new /obj/item/weapon/spacecash/bundle/c1000(src)
 	new /obj/item/weapon/spacecash/bundle/c1000(src)
+
+/obj/item/weapon/storage/box/syndie_kit/gun_mods
+	name = "Gun Mods Kit"
+	desc = "Gun Mods Kit"
+
+/obj/item/weapon/storage/box/syndie_kit/gun_mods/populate_contents()
+	for(var/i in subtypesof(/obj/item/weapon/gun_upgrade))
+		var/obj/test = i
+		if(initial(test.icon_state))
+			new i(src)
```

Comment:
This is now unobtainable, and could be removed.

Reply:
Done